src/command_modules/azure-cli-sql/HISTORY.rst

```diff
@@ -0,0 +1,275 @@
+.. :changelog:
+
+Release History
```

Comment:
Why do you need such a HISTORY.rst file here?

Reply:
it was from the old branch. I have removed it.